src/app/manage-employees/manage-employees.component.css

```diff
@@ -0,0 +1,59 @@
+body {
+    margin: 0;
+}
+
+.manage-employees-container {
+    width: 100vw;
+    height: 100vh;
+    display: grid;
+
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 50px 1fr 1fr 100px;
+
+    gap: 10px;
+
+    padding: 10px;
+    box-sizing: border-box;
+
+}
+
+.border {
+    padding: 10px;
+    border: 1px solid black;
+}
+
+.header {
+    grid-column-start: 1;
+    grid-column-end: 5;
+}
+
+.content-large {
+    grid-row-start: 2;
+    grid-row-end: span 2;
+    grid-column-start: 2;
+    grid-column-end: 5;
+}
+
+.content-large.scroll {
+    overflow: hidden;
+    overflow-y: scroll;
+}
+
+.content-small {
+    grid-column-start: 1;
+    grid-column-end: 2;
+}
+
+.content-smallest {
+    grid-column-start: 1;
+    grid-column-end: 1;
+}
+
+.footer {
+    grid-column-start: 1;
+    grid-column-end: 5;
+}
+
+.sort-departments {
+    padding-left: 20px;
+}
```

Comment:
This should still be converted to tailwind,  like the other file you did perfectly.
Anything you can do with css you can do in tailwind aswell.